server/models/processedTxs.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose')
+
+let processedTxsSchema = mongoose.Schema({
+  transactionHash: String,
+  logIndex: String
+})
+
+processedTxsSchema.index({transactionHash: 1, logIndex: 1},  {unique: true})
```

Comment:
Would be good to go through the app code, see what other params you query by, and add indexes for them